scripts/armv7a9-zynq7000.sh

```diff
@@ -1,20 +1,30 @@
 #!/bin/bash
 #
-# Shell script for running phoenix-rtos loader on QEMU (zynq-7000)
+# Shell script for running phoenix-rtos loader on Qemu fork from Xilinx (zynq-7000)
 #
 # Copyright 2021 Phoenix Systems
 # Author: Hubert Buczynski
 #
 
-IMG_ZYNQ7000="$(dirname "${BASH_SOURCE[0]}")/../_boot/plo-ram-armv7a9-zynq7000.img"
+IMG_PLO_ZYNQ7000="$(dirname "${BASH_SOURCE[0]}")/../_boot/plo-armv7a9-zynq7000-qemu.img"
+IMG_FLASH_QEMU="$(dirname "${BASH_SOURCE[0]}")/../_build/armv7a9-zynq7000/prog/flash-armv7a9-zynq7000.bin"
+IMG_PHOENIX_ZYNQ7000="$(dirname "${BASH_SOURCE[0]}")/../_boot/phoenix-armv7a9-zynq7000.disk"
+DTB_ZYNQ7000="$(dirname "${BASH_SOURCE[0]}")/../scripts/zynq7000-zc702.dtb"
 
-if [ ! -f "$IMG_ZYNQ7000" ]; then
-	echo "File $IMG_ZYNQ7000 does not exist"
+if [[ ! -f "$IMG_PLO_ZYNQ7000" && ! -f "$IMG_PHOENIX_ZYNQ7000" && ! -f "$DTB_ZYNQ7000" ]]; then
```

Comment:
I assume you meant `||` (right now this will be false if only 1 file is missing).

I would go for checking inside loop / function for easier understanding, eg.:

```bash
for FILE in "$IMG_PLO_ZYNQ7000" "$IMG_PHOENIX_ZYNQ7000" "$DTB_ZYNQ7000"; do
  if [ ! -f "$FILE" ]; then
    echo "Missing required file: $FILE"
    exit 1
  fi
done
```
```